### PR TITLE
Use `Steps` component for `add-content-collections.mdx`

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

Use Starlight's `Steps` component for `add-content-collections.mdx`

Docs see https://starlight.astro.build/guides/components/#steps

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by at-the-vr — COMMENTED** [review]
I find it unusual seeing a step prefixed by a heading in the preview 😕 but overall changes are 👍  

image

**Comment by sarah11918** [comment]
Wow, this looks great! I love our steps! Thanks for starting the ball rolling with this @liruifengv :raised_hands: 

So, I can say that headings are required to generate a table of contents. I think it's helpful to have that there for people to see, and for more direct linking to parts of the process.

If it seems weird to have a heading then, like, Step 7, it's because I like keeping the numbers going to help people know where in the whole process they are.

If you're thinking "maybe there should be some text after a heading"... well, I don't necessarily want to add more text if it's unnecessary. It's just more to read and slows people down if the steps are clear enough.

I will note that we *currently* do have these numbers right after the headings, so all this PR does is make them Fancy Steps vs an ordered list. So, we can discuss in another issue/PR whether we *want* to add content, but this is literally NWTWWHB! (And, it's better!)

image

File: src/content/docs/en/tutorials/add-content-collections.mdx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import Box from '~/components/tutorial/Box.astro';
 import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
 import PreCheck from '~/components/tutorial/PreCheck.astro';
 import Option from '~/components/tutorial/Option.astro';
+import { Steps } from '@astrojs/starlight/components'
```
Comment (sarah11918) [inline src/content/docs/en/tutorials/add-content-collections.mdx:12]:
```suggestion
import { Steps } from '@astrojs/starlight/components';
```
I'm saving @alexnguyennz a PR! :sweat_smile:

**Review by sarah11918 — APPROVED** [review]
This looks great to me! Would like an approval from @delucis for this first usage of Steps in docs, but I'm happy!

**Review by delucis — APPROVED** [review]
Usage looks good to me! Thanks for kicking off the steps revolution @liruifengv 🚀

**Review by ElianCodes — APPROVED** [review]
This is amazing!

**Comment by sarah11918** [comment]
Reminder this is yours to merge when you're happy, @liruifengv !

**Comment by liruifengv** [comment]
!coauthor

**Comment by liruifengv** [comment]
Ship it! 🚀

**Pull request merged**